thorlcr/thorutil/thbuf.cpp

```diff
@@ -1889,13 +1913,16 @@ class CSharedFullSpillingWriteAhead : public CInterfaceOf<ISharedRowStreamReader
     }
     void closeWriter()
     {
+        iFileIO->flush();
```

Comment:
I think it is safe (as guarded in 2 places), but not in allStopped handling.
It would look safer to common up the if (outputStream) check here in closeWriter (remove from dtor and reset()